subcommands/targets/list.go

```diff
@@ -30,6 +30,7 @@ type targetListing struct {
 	hardwareIds []string
 	tags        []string
 	apps        []string
+	origin      string
 }
```

Comment:
What do you think about other TufCustom fields (target format, 3 hashes)?
Does it make sense to add these columns as optional values ?

Reply:
probably not. target-format != ostree don't get skipped. The "sha" fields are really long git urls and wouldn't fit well in column style list

Reply:
but - why not have the "sha" fields optional. saves a support ticket some day. i'll add it.

Reply:
https://github.com/foundriesio/fioctl/pull/126/commits/b4a98a43a71ae62f5fc27f477c0e261ec4ec81f5